docs/infrastructure/eks-quickstart.md

```diff
@@ -1,204 +1,159 @@
 
-## TL;DR
+## 2. Set up the Terraform state storage backend
 
-### Setting up an EKS cluster for Codacy
+The [backend](https://www.terraform.io/docs/backends/index.html) stores the current and historical state of your infrastructure.
 
-Assuming **AWS is well configured** and you fulfill all the prerequisites,
-the setup without state storage, can be done with:
+Although using the backend is optional, we recommend that you deploy it, particularly if you are planning to use these templates to make modifications to the cluster in the future:
 
-```bash
-$ export AWS_SDK_LOAD_CONFIG=1
-$ cd main/
-$ terraform init && terraform apply # should take around 15 mins
-$ cd ../setup/
-$ terraform init && terraform apply
-$ aws eks update-kubeconfig --name codacy-cluster
-```
+1. Initialize Terraform and apply the changes inside the `backend/` directory, then follow Terraform's instructions:
 
-## Deployment - Long version
+    ```bash
+    cd backend/
+    terraform init && terraform apply
+    ```
 
-### 1. `backend` - setup terraform state storage
+    An Amazon S3 bucket with a unique name to save the infrastructure state is created.
 
-The [backend](https://www.terraform.io/docs/backends/index.html) stores
-the current and historical state of your infrastructure. Deploying and using
-it is optional, but it might make your life easier if you are planning to use
-these templates to make modifications to the cluster in the future.
+1. Note the value of `state_bucket_name` in the output of the command.
 
-To deploy it run:
+1. Edit the `config.tf` files that exist in the `main/` and `setup/` directories and follow the instructions to set the name of the Amazon S3 bucket and to enable the use of the backend in those files.
 
-```bash
-terraform init && terraform apply
-```
+## 3. Create a vanilla Amazon EKS cluster
 
-inside the `backend/` folder and follow terraform's instructions.
+Create a cluster that includes all the required network and node setup:
 
-An S3 bucket with a unique name to save your state will be created. Note the value of `state_bucket_name` in the output of the command and set it on the `config.tf` file of the `main/` and `setup/` stacks where indicated.
+1. Initialize Terraform and apply the changes inside the `main/` directory, then follow Terraform's instructions:
 
-### 2. `main` - create a vanilla EKS cluster
+    ```bash
+    cd ../main/
+    terraform init && terraform apply
+    ```
 
-To create a cluster, along with all the necessary network and nodes setup
-it requires:
+    This process takes around 10 minutes.
 
-```bash
-terraform init && terraform apply
-```
+1. Consider if you want to tailor the cluster to your needs by customizing the cluster configuration.
 
-inside the `main/` folder and follow terraform's instructions. This takes a while (~10min).
+    The cluster configuration (such as the type and number of nodes, network CIDRs, etc.) is exposed as variables in the `main/variables.tf` file.
```

Comment:
Should we add here a link the the variables.tf file?

https://github.com/codacy/chart/blob/master/docs/infrastructure/EKS/main/variables.tf

Reply:
I prefer avoiding doing that because the user will have to edit the file on their local copy of the repository (so they'll have to find it in the filesystem). It's also less trouble to keep the links up-to-date and working.

Reply:
Thinking again, this is not a file that the user needs to edit, they only need to read it. So adding a link can make sense, yes!